app/routes/photos/components/GalleryPictureModal.js

```diff
@@ -181,12 +181,9 @@ export default class GalleryPictureModal extends Component<Props, State> {
     }
   };
 
-  handleSwipeRight = () => {
-    this.previousGalleryPicture();
-  };
-
-  handleSwipeLeft = () => {
-    this.nextGalleryPicture();
+  handleSwipe = ({ dir }: RIGHT | LEFT) => {
```

Comment:
Does this make sense? `dir` is inside an object og type ` RIGHT | LEFT`, and then compared to `RIGHT` or `LEFT` 🤔

Reply:
Yeah, that's wrong 🙃

Reply:
Ahh. Then it makes some more sense! Replacing `RIGHT | LEFT` with `EventData` (if the flow type exists), it should be ok. (or even `{ dir } : { dir: RIGHT | LEFT | UP | DOWN }`)

Reply:
The flow type didn't exist, so I went with the latter approach.